src/frontend/src/controllers/API/queries/files/use-download-files.ts

```diff
@@ -20,7 +20,11 @@ export const useGetDownloadFileMutation: useMutationFunctionType<
   const getDownloadImagesFn = async () => {
     if (!params) return;
     // need to use fetch because axios convert blob data to string, and this convertion can corrupt the file
-    const response = await fetch(`${getURL("FILES")}/download/${params.path}`);
+    const response = await fetch(`${getURL("FILES")}/download/${params.path}`, {
+      headers: {
+        Accept: "*/*",
+      },
+    });
```

Comment:
does the problem always happen? can we not make the request if we know the problem might occur?